assets/elasticsearch/ingest.py

```
@@ -0,0 +1,93 @@
+        else:
+            logging.error(f"Failed to index document at {count}: {info}")
+        if count % 100000 == 0:
+            logging.info(f"Index {count} logs")
+
+    requests.post(f"http://localhost:9202/{collection_name}/_flush/")
+
+
+if __name__ == "__main__":
+    try:
+        ingest_dataset()
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)